bsip-0056.md

```diff
@@ -0,0 +1,314 @@
+    BSIP: T.B.D.
+    Title: Managed vesting balances
+    Authors: Fabian Schuh <Fabian.Schuh@BlockchainProjectsBV.com>
+             Stefan Schießl <Stefan.Schiessl@BlockchainProjectsBV.com>
+    Status: Draft
+    Type: Protocol
+    Created: 2018-10-10
+    Discussion: <url>
+    Worker: <Id of worker proposal> (optional)
```

Comment:
Consider removing this for the "draft" stage. We can edit this if/when the voters approve the BSIP and it moves forward with implementation. Alternatively, may leave the line, but change it to "<TBD>".